### PR TITLE
Improve code style and do some minor fix in scripting

### PR DESCRIPTION
**Pull request by PragmaTwice**
Related to almost all functions in scripting.

### CODE REVIEW

**Review by torwig — DISMISSED** [review]

File: src/storage/scripting.cc [hunk 1]
```diff
@@ -109,12 +110,10 @@ void loadFuncs(lua_State *lua, bool read_only) {
...
-  if (read_only) {
-    /* redis.read_only */
-    lua_pushstring(lua, "read_only");
-    lua_pushboolean(lua, 1);
-    lua_settable(lua, -3);
-  }
+  /* redis.read_only */
+  lua_pushstring(lua, "read_only");
+  lua_pushboolean(lua, read_only);
+  lua_settable(lua, -3);
```
Comment (PragmaTwice) [inline src/storage/scripting.cc:116]: It is not intuitionistic if the boolean value only exists when read_only is true

File: src/storage/scripting.cc [hunk 2]
```diff
@@ -175,26 +174,20 @@ int redisLogCommand(lua_State *lua) {
...
   }
-  level = static_cast<int>(lua_tonumber(lua, -argc));
+  int level = static_cast<int>(lua_tonumber(lua, -argc));
   if (level < LL_DEBUG || level > LL_WARNING) {
     lua_pushstring(lua, "Invalid debug level.");
     return lua_error(lua);
   }
-  if (level < GetServer()->GetConfig()->log_level) {
-    return 0;
-  }
```
Comment (PragmaTwice) [inline src/storage/scripting.cc:185]: This level cannot be compared with loglevel in config.

File: src/storage/scripting.cc [hunk 3]
```diff
@@ -306,48 +294,54 @@ Status evalGenericCommand(Redis::Connection *conn, const std::vector<std::string
...
   auto commands = Redis::GetCommands();
   auto cmd_iter = commands->find(Util::ToLower(args[0]));
   if (cmd_iter == commands->end()) {
     pushError(lua, "Unknown Redis command called from Lua script");
     return raise_error ? raiseError(lua) : 1;
   }
+
   auto redisCmd = cmd_iter->second;
-  if (read_only && redisCmd->is_write()) {
+  if (read_only && ~(redisCmd->flags & Redis::kCmdReadOnly)) {
```
Comment (PragmaTwice) [inline src/storage/scripting.cc]: There is no assumption that a command is read-only if it has no write flag

File: src/storage/scripting.cc [hunk 4]
```diff
@@ -260,8 +247,9 @@ Status evalGenericCommand(Redis::Connection *conn, const std::vector<std::string
     } else {
       body = args[1];
     }
+
     std::string sha;
-    auto s = createFunction(srv, body, &sha, lua);
+    auto s = createFunction(srv, body, &sha, lua, false);
```
Comment (PragmaTwice) [inline src/storage/scripting.cc:252]: We do not need to store the script body here.

**Review by git-hulk — DISMISSED** [review]

**Review by torwig — DISMISSED** [review]

**Review by git-hulk — APPROVED** [review]

**Pull request merged**